### PR TITLE
Corrected Lines for Tyla Monastery Missions

### PR DESCRIPTION
**Pull request by ttgmichael**
https://forum.treeofsavior.com/t/different-translations-for-a-same-quest/121316

Asked to fix the Quest Title only, but I figured I'd check the quest instructions as a whole for the heck of it xD.

Please let me know if I missed something/typo (like the Woodluwa korean typo) :P

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -8590,140 +8590,140 @@ QUEST_LV_0200_20150714_008593	It seems that Hunter Reina is still not feeling we
...
-QUEST_LV_0200_20150714_008632	You've collected lots of beeswaxes. Hand them over to Chad.
+QUEST_LV_0200_20150714_008597	$Hunter Natasha wants to persuade Senior Monk Marko, but it's not that easy. Listen to what she says.
+QUEST_LV_0200_20150714_008598	$Talk to Senior Monk Marko
+QUEST_LV_0200_20150714_008599	$Natasha wants you to persuade Marko on behalf of her. Talk to Marko.
+QUEST_LV_0200_20150714_008600	$Marko doesnt trust the Hunters. Pass the words you heard to Natasha.
+QUEST_LV_0200_20150714_008601	$Senior Monk Marko is trying his best to retake the Monastery. Let Marko know that the Hunters can help in retake the Monastery.
+QUEST_LV_0200_20150714_008602	$Talk to the monks
+QUEST_LV_0200_20150714_008603	$Marko wants you to tell the monks who were sent to the lower area that the Hunters are participating. Meet the monks and talk to them.
+QUEST_LV_0200_20150714_008604	$A young monk has collapsed, causing Marko to be very nervous. Hunter Natasha may know a way to improve the status of the young monk.
+QUEST_LV_0200_20150714_008605	$The Unknown Poison
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:8601]: Unknown Poison

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -8925,7 +8925,7 @@ QUEST_LV_0200_20150908_008928	Freedom at last...
 QUEST_LV_0200_20150908_008929	Finally...
 QUEST_LV_0200_20150908_008930	I'm free from the restraint...
 QUEST_LV_0200_20150908_008931	{memo X}This is enough...
-QUEST_LV_0200_20150908_008932	Freedom. Finally, free...
+QUEST_LV_0200_20150908_008932	I'm free. Finally, I'm free...
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Original does say 'freedom'.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**